schemas/schemas/infoQuestion.json

```diff
@@ -77,13 +77,25 @@
             "type": "string"
         },
         "gradingMethod": {
-            "description": "The grading method used for this question.",
+            "description": "The grading method used for this question. (DEPRECATED - Use `gradingMethods` below)",
```

Comment:
Personally, I don't think we ever actually want to deprecate "gradingMethod" because for a huge amount of use cases, people will only want their question to have one grading method, and they won't want to specify a list of grading methods.